readme.md

```diff
@@ -34,16 +34,16 @@ gulp.task('default', function () {
 
 ##### ui
 
-Type: `string`  
```

Comment:
For readability. Double spaces in Markdown will be converted to `<br>`.

[This rule is explicitly declared in our `.editorconfig`](https://github.com/sindresorhus/gulp-mocha/blob/80781186938661a6cc024536c2fa8ae05a9cb9ea/.editorconfig#L14-L15). You should use [EditorConfig](http://editorconfig.org/) plugin for your editor when you contribute to the project which includes an `.editorconfig` file.